consensus/scheme/rolldpos/endorsementmanager.go

```diff
@@ -368,16 +368,13 @@ func (m *endorsementManager) AddVoteEndorsement(
 }
 
 func (m *endorsementManager) Cleanup(timestamp time.Time) error {
-	cleaned := &endorsementManager{
-		eManagerDB:  m.eManagerDB,
-		collections: map[string]*blockEndorsementCollection{},
-	}
+	cleanedCollections := map[string]*blockEndorsementCollection{}
```

Comment:
could you just use the change in
https://github.com/iotexproject/iotex-core/commit/f491a23a77c0e47a2ea92344b189b354592835ab

that is what's being tested on nightly-0 now

Reply:
oh ok !!